packages/MonorepoBuilder/packages/Split/tests/Process/ProcessFactoryTest.php

```diff
@@ -27,7 +27,15 @@ public function test(): void
         );
 
         $subsplitRealpath = realpath(__DIR__ . '/../../bash/subsplit.sh');
-        $commandLine = "'" . $subsplitRealpath . "' '--from-directory=localDirectory' '--to-repository=git@github.com:Symplify/Symplify.git' '--branch=master' %s '--repository=%s/.git'";
-        $this->assertStringMatchesFormat($commandLine, $subsplitProcess->getCommandLine());
+        $this->assertRegExp(
+            implode('', [
+                "/^['\"]?" . preg_quote($subsplitRealpath, '/') . "['\"]? ",
+                "['\"]?\-\-from\-directory\=localDirectory['\"]? ",
+                "['\"]?\-\-to\-repository\=git@github\.com\:Symplify\/Symplify\.git['\"]? ",
+                "['\"]?\-\-branch\=master['\"]? [^\r\n]+ ",
+                "['\"]?\-\-repository\=[^\r\n]+\/\.git['\"]?$/s",
+            ]),
+            $subsplitProcess->getCommandLine()
+        );
```

Comment:
Why not 2 lines, one for Windows and one for *Unix? I have no idea how I'd debug this

Reply:
I did not find a assert function which accepts more than one format string.
An alternative would be, to create a separate test group for Linux/Windows and only execute the tests for the current OS.

Reply:
Depending on the path used, both on Windows and Linux the command and/or the arguments can be escaped in either single or double quotes. Thats gives lots of alternatives.

The given expression is the result of the former format string (which will simply be mapped to a regular expression internally) and salted with optional single/double quotes around each argument.